src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -253,7 +254,9 @@ private static string BuildApi(DocsEnvironment? environment = null)
             return (environment ?? DocsEnvironment) switch
             {
                 DocsEnvironment.Prod => "https://buildapi.docs.microsoft.com",
-                _ => "https://BuildApiPubDev.azurefd.net",
+                DocsEnvironment.PPE => "https://BuildApiPubDev.azurefd.net",
+                DocsEnvironment.Perf => "https://op-build-test.azurewebsites.net",
+                _ => throw new NotSupportedException(),
```

Comment:
throw `InvalidOperationException` like the others

Reply:
Done.